templates/extras/simp.md

```diff
@@ -27,15 +39,15 @@ begin
 end
 ```
 
-How would a human solve that goal? They would notice that `a * a⁻¹ =
-1`, that `1 * 1 = 1` and so on, until they had simplified the example
+How would a human solve that goal? They would notice that `a * a⁻¹ = 1`,
+that `1 * 1 = 1`, and so on, until they had simplified the example
 to `b = b`, which is obviously true.
 
 This is also what the simplifier is doing. Indeed, if you add
```

Comment:
Could I suggest you edit this paragraph so it introduces `simp?` first, and then mentions that `set_option` possibilities if you need even more detail? I think training people to use `simp?` is good. :-)

Reply:
I added the output of `set_option trace.simplify.rewrite true` along with what `simp?` suggests to illustrate the meaning of `simp?`.